<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiRepeater</class>
 <widget class="QWidget" name="GuiRepeater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="sendRequestButton">
     <property name="text">
      <string>Send Request</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="requestVLayout">
     <item>
      <layout class="QGridLayout" name="editRequestLayout">
       <item row="5" column="0">
        <widget class="QPlainTextEdit" name="requestBodyPlainTextEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="requestBodyLabel">
         <property name="text">
          <string>Request Body</string>
         </property>
         <property name="buddy">
          <cstring>requestBodyPlainTextEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QLineEdit" name="urlLineEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="urlLineEditLabel">
           <property name="text">
            <string>URL</string>
           </property>
           <property name="buddy">
            <cstring>urlLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="requestMethodLabel">
           <property name="text">
            <string>Method</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="requestMethodcomboBox">
           <property name="currentText">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="replyStatusLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="replyReadOnlyplainTextEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
